### PR TITLE
add dragonwell support

### PR DESCRIPTION
**Pull request by gdams**

### CODE REVIEW

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 1]
```diff
@@ -135,8 +138,10 @@ class Build {
             suffix="corretto/corretto-${javaNumber}"
         } else if (buildConfig.VARIANT == "openj9") {
```
Comment (karianna) [inline pipelines/build/common/openjdk_build_pipeline.groovy:139]: Hmm we need to refactor this overloaded term VARIANT should probably refer to the underlying VM and we should introduce PROVIDER for 'who is behind this thing'

File: pipelines/build/dockerFiles/dragonwell.dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+ARG image
+
+FROM $image
+
+RUN \
+    # Dragonewell 8 requires a dragonwell 8 BootJDK
```
Comment (karianna) [inline pipelines/build/dockerFiles/dragonwell.dockerfile:6]: So we're only doing 8 for now?

File: pipelines/jobs/configurations/jdk8u.groovy [hunk 3]
```diff
@@ -6,7 +6,8 @@ targetConfigurations = [
         "x64Linux"      : [
                 "hotspot",
                 "openj9",
-                "corretto"
+                "corretto",
+                "alibiba"
```
Comment (karianna) [inline pipelines/jobs/configurations/jdk8u.groovy:10]: should be called dragonwell?

**Review by karianna — CHANGES_REQUESTED** [review]

File: pipelines/jobs/configurations/jdk8u.groovy (hunk 3 above)
Comment (smlambert) [reply pipelines/jobs/configurations/jdk8u.groovy:10]: I think so, also alibaba is spelled wrong here

File: pipelines/build/dockerFiles/dragonwell.dockerfile (hunk 2 above)
Comment (gdams) [reply pipelines/build/dockerFiles/dragonwell.dockerfile:6]: Dragonwell 11 can be built with a regular bootJDK. This is specific to 8

File: pipelines/build/common/openjdk_build_pipeline.groovy (hunk 1 above)
Comment (gdams) [reply pipelines/build/common/openjdk_build_pipeline.groovy:139]: agreed, we can do some logic at the publish phase to control this for now.

**Review by karianna — APPROVED** [review]

**Comment by karianna** [comment]
run tests

File: pipelines/build/common/openjdk_build_pipeline.groovy (hunk 1 above)
Comment (gdams) [reply pipelines/build/common/openjdk_build_pipeline.groovy:139]: PR to push these binaries to a separate repo for the API: https://github.com/AdoptOpenJDK/openjdk-website-backend/pull/72/files

**Review by smlambert — APPROVED** [review]
Thanks @gdams !

**Pull request merged**